src/tallies/tally_filter_zernike.F90

```diff
@@ -67,18 +80,20 @@ subroutine get_all_bins(this, p, estimator, match)
     x = p % coord(1) % xyz(1) - this % x
     y = p % coord(1) % xyz(2) - this % y
     r = sqrt(x*x + y*y)/this % r
-    theta = atan2(y, x)
-
-    ! Get moments for Zernike polynomial orders 0..n
-    call calc_zn(this % order, r, theta, zn)
-
-    do i = 1, this % n_bins
-      call match % bins % push_back(i)
-      call match % weights % push_back(zn(i))
-    end do
-  end subroutine get_all_bins
-
-  subroutine to_statepoint(this, filter_group)
+    if (r <= 1) then
```

Comment:
Use `ONE` rather than `1`.  Fortran can give some annoying errors when you mix numbers of different precisions, and `1` is an `integer` whereas `ONE` is a `real(8)`